tools/nd_freeze

```diff
@@ -314,8 +314,9 @@ sub write_snapshot_sources {
         my $line_prefix = "";
         $line_prefix = "#" if ($sources{$key}{Label} eq 'Debian-Security');
 
-        print $fp "${line_prefix}deb http://${domain}/archive/${label}/${next_timestamp}/ ${sources{$key}{Codename}} ${sources{$key}{Component}}\n";
-        logg("debug", "Adding 'deb http://${domain}/archive/${label}/${next_timestamp}/ ${sources{$key}{Codename}} ${sources{$key}{Component}}' to $snapshots_sources_file");
+        my $trusted_option = ( is_minimum_apt_version("0.9.40") ? "[trusted=yes]" : "" );
```

Comment:
Based on what I remember from the call last Thursday, I'm surprised this is 0.9.40 rather than 0.9.0, which is the first release that contains 4b42f43bed369817398b6c8d538f08e5bf6dff76, the commit that added support for `trusted=yes`.

Reply:
You're right. I had that version in my notes which are wrong. I went back and verified that commit number with "git log -Strusted=yes" and ran it against "git tag --contains ..." to see (again) that 0.9.0 is the release to go with.